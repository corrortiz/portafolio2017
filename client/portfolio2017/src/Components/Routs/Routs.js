// @flow
import React from 'react';
import { Route, Switch, withRouter } from 'react-router-dom';

import { TransitionGroup, CSSTransition } from 'react-transition-group';

import Contact from './ContactRout/ContactRout';
import Home from './HomeRoute/HomeRoute';
import Projects from './ProjectsRoute/ProjectsRoute';
import Services from './ServicesRoute/ServicesRoute';

const Routs = withRouter(({ location }) => (
  <div>
    <TransitionGroup>
      <CSSTransition key={location.key} classNames="animate" timeout={350}>
        <Switch className="Routs" location={location}>
          <Route exact path="/" component={Home} />
          <Route exact path="/services" component={Services} />
          <Route exact path="/projects" component={Projects} />
          <Route exact path="/contact" component={Contact} />
        </Switch>
      </CSSTransition>
    </TransitionGroup>
  </div>
));
export default Routs;
