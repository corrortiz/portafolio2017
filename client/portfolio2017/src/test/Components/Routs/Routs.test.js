// @flow
import React from 'react';
import { shallow } from 'enzyme';
import Routs from '../../../Components/Routs/Routs';

it('should compare a snapshot of Routs', () => {
  const wrapper = shallow(<Routs />);
  expect(wrapper).toMatchSnapshot();
});
