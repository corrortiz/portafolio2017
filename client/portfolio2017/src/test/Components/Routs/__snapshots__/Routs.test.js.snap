// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should compare a snapshot of Routs 1`] = `
<Route
  render={[Function]}
/>
`;
